libs/components/forms/src/lib/modules/selection-box/selection-box.component.ts

```diff
@@ -126,22 +131,6 @@ export class SkySelectionBoxComponent implements OnDestroy {
     this.#ngUnsubscribe.complete();
   }
 
-  /**
-   * Since we are programatically firing a click on the control,
-   * make sure user is not clicking on the control before firing click logic.
-   */
-  public onClick(event: any): void {
```

Comment:
The onClick event is no longer needed since native JavaScript behavior for `label` elements is to activate the child checkbox/radio.